### ISSUE TITLE
All effects and solid colours not talking to LEDs correctly

### ISSUE THREAD

**Issue by pbezza** [Aircoookie/WLED#1418]
I have installed the v11.0 bin on to my QuinLED-Dig-Quad with an Esp 32 but its not talking to the LEDs correctly. Choosing peek in the GUI is also wrong and matches what my LEDs are showing. I have 6 5m lengths of ws2812b 50 per length. The first 54 LEDs barely show any colour and the last 50 show nothing. If I add more LEDs, they also show nothing. As the peek is identical to the physical LEDs, I am positive that my LEDs are not at fault. I have tried swapping out these LEDs but yielded the same results.
![Screenshot_2020-12-02-21-08-28-356_com android chrome](https://user-images.githubusercontent.com/34596500/100861074-716c2500-34e5-11eb-85f6-529d89373de7.jpg) 
![Screenshot_2020-12-02-21-08-20-907_com android chrome](https://user-images.githubusercontent.com/34596500/100861221-a0829680-34e5-11eb-8b50-7b438c370b96.jpg)

**Comment by pbezza** [comment]
Tried Quindor's v0.11.0 bin files but have the same problem. Switching to an older version to see if the problem goes away.

**Comment by Aircoookie** [comment]
Hi, sorry for the issue! Others have had this problem as well, it can happen if some flash contents are invalid.
It often helps to erase the flash completely (for example with `esptool.py erase_flash`, your flasher might also have such an option) and then reinstall the binary.

**Comment by pbezza** [comment]
Thanks for replying. I'll try tonight after work. It was a brand new esp32 which should have been blank already.

**Comment by sansillusion** [comment]
Inhave had weird issues mainly stability, I flashed multiple versions and multiple options activated/deactivated etc... Used usb and ota, when using USB I often chose erase data option (having to reconfig wifi, leds etc...).

Lately I wanted to flash a bin (before 0.11) and looked at the quickstart guide. I decided to flash the provided bootloader, when it rebooted I noticed it had reset everything. So I reconfigured everything before trying to flash the bin. Weirdly with the save wled I started havin 190kb free mem instead of 170kb I decided to keep it like that and it's been working great ever since. I compiled latest source a few days ago and it worked like a charm without any bugs anymore ! Could the bootloader from Aircoookie be somehow better than what my Arduino IDE was giving me (esp32 intalled trough git master aswell as wled, except since 0.11.0 wich I had to install and use vscode/pio)
